packages/appsync-emulator-serverless/serverCore.js

```diff
@@ -249,13 +249,14 @@ const executeGQL = async ({ schema, documentAST, jwt, variables }) => {
 };
 
 const createGQLHandler = ({ schema, subServer }) => async (req, res) => {
-  const {
-    headers: { authorization = null },
-  } = req;
-  if (!authorization) {
+  // const { headers: { authorization = null, }, } = req;
+  const { headers } = req;
+  log.info('req', { req });
+
+  if (!headers.authorization && !headers['x-api-key']) {
     throw new Error('Must pass authorization header');
   }
-  const jwt = jwtDecode(authorization);
+  const jwt = headers.authorization ? jwtDecode(headers.authorization) : {};
```

Comment:
Hmm, thinking it would be useful to be able to access user information within appsync handlers for folks who want to use API key authentication.  Perhaps we could emulate that somehow?

Reply:
Does AWS provide user info for API key user's? 🤔